src/components/Timer/Timer.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import styles from './Timer.module.css';
+
+export class Timer extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            seconds: 30
```

Comment:
I think the next step here is that when the server prompts the players, it should decide what the timer starts at, otherwise we will have inconsistencies

Reply:
okay I might add that as another issue

Reply:
I think it should probably be done in this pr, I can help get you started